onnxruntime/core/providers/openvino/ov_interface.h

```diff
@@ -0,0 +1,114 @@
+// Copyright(C) 2019 Intel Corporation
+// Licensed under the MIT License
+
+#pragma once
+
+#include <inference_engine.hpp>
+#include <openvino/core/core.hpp>
+#include <openvino/runtime/runtime.hpp>
+
+namespace onnxruntime {
+
+namespace openvino_ep {
+
+class ov_core;
+class ov_infer_request; 
+class ov_exe_network; 
+ 
+    #if defined (OPENVINO_2022_1) 
```

Comment:
I somehow feel the code is started from towards very right. Probably too many tabs. Can we stick to left few less tabs.
<The entire file follows same flow>